torch/distributed/nn/api/remote_module.py

```diff
@@ -143,6 +159,114 @@ def __init__(
             method = torch.jit.export(method)
             setattr(self, method_name, types.MethodType(method, self))
 
+    def _raise_not_supported(self, name):
+        raise ValueError("Method ``{}`` not supported for RemoteModule".format(name))
```

Comment:
Let's add a simple unit in `remote_module_test.py` that validates (using self.assertRaisesRegex) all of these throw appropriate exceptions.